src/InternalCommand.roc

```diff
@@ -0,0 +1,24 @@
+interface InternalCommand
+    exposes [
+        Command,
+        Output,
+        CommandErr,
+    ]
+    imports []
+
+CommandErr : [
+    ExitStatus I32,
+    IOError Str,
+]
+
+Command : {
+    program : Str,
+    args : List Str, # [arg0, arg1, arg2, arg3, ...]
+    envs : List Str, # [key0, value0, key1, value1, key2, value2, ...]
```

Comment:
Any reason not to prefer `List (Str, Str)` here?

Reply:
I hadn't thought of that. 

I've attempted to change it to this, but it appears List (Str, Str) isn't supported in glue. I'll log an issue.